spec/controllers/videos_controller_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "spec_helper"
+
+describe VideosController do
+  describe "GET show" do
+    it "sets @video for authenticated user" do
+      session[:id] = Fabricate(:user).id
+      video = Fabricate(:video)
+      get :show, id: video.id
+      expect(assigns(:video)).to eq(video)
+    end
+
+    it "sets @reviews for authenticated users" do
+      session[:id] = Fabricate(:user).id
+      video = Fabricate(:video)
+      review1 = Fabricate(:review, video: video)
+      review2 = Fabricate(:review, video: video)
+      get :show, id: video.id
+      assigns(:reviews).should =~ [review1, review2]
```

Comment:
You are trying 2 different RSpec syntax styles, this is `should` syntax and elsewhere `expect` syntax style. But proper way is to stick and use 1 way, if you want to match then `match_array` matcher should be used.